src/ISA/ISA/JsonTypes/URI.fs

```diff
@@ -1,13 +1,3 @@
 ﻿namespace ISA
 
-type URI = string
-
-module URI =
-
-    /// Create a ISAJson URI from a ISATab string entry
-    let fromString (s : string) : URI =
-        s
-
-    /// Create a ISATab string URI from a ISAJson object
-    let toString (s : URI) : string =
-        s
\ No newline at end of file
+type URI = string
```

Comment:
I used these "to" and "from" functions for later checks against the validity of the URI. Not sure whether to keep them or forget the whole ordeal as many places where "URI" is used in ISA you wouldn't necessarily expect a URI.

Reply:
I think i do not understand. These functions did not do anything, right? Convert `URI` to `string` and reverse, but  they are already the same, because `type URI = string`.

Reply:
It's a "TO-DO" kind of thing. The validity checks weren't yet implemented.